html/changelogs/.all_changelog.yml

```diff
@@ -2598,7 +2598,7 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - tweak: Replaced most of the references of the Nyx system with Tau Ceti.
   - tweak: Voidsuits, armor and armored uniforms should have a better resistance to
       taser and baton hits.
-  - tweak: Head of security armor's options were tweaked to be more like each other.
```

Comment:
You don't need to change old changelogs.

Reply:
I have no idea how this got changed. Lmao.